lib/tpm2_hmac_auth.h

```diff
@@ -0,0 +1,40 @@
+//**********************************************************************;
+// Copyright (c) 2015-2018, Intel Corporation
```

Comment:
new file, this will apply throughout. No 2015 here.

Reply:
Fixed